Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
new file mode 100644
--- /dev/null	(revision 7f605a2d36df4c6f50f317db9cee3f629986460e)
+++ b/.idea/vcs.xml	(revision 7f605a2d36df4c6f50f317db9cee3f629986460e)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: lab4tab/zad2.py
===================================================================
diff --git a/lab4tab/zad2.py b/lab4tab/zad2.py
--- a/lab4tab/zad2.py	(revision 68dacdb6817dd1b1abf5e1dcdaef7e666e61a92d)
+++ b/lab4tab/zad2.py	(revision 7f605a2d36df4c6f50f317db9cee3f629986460e)
@@ -9,11 +9,14 @@
 • Utwórz listę zestaw_1_2 będącą złączeniem wartości z list zestaw_1 oraz zestaw_2. Posortuj i wyświetl
 listę'''
 import random
-
-x=random.randint(1,10)
-zestaw_1=[]
-u=int(input("jak długa lista"))
+zestaw_1 = []
+n = int(input("Podaj rozmiar lista: "))
+for i in range(n):
+    x = random.randint(1, 10)
+    zestaw_1.append(x)
+    print(zestaw_1)
 
-zestaw_1
-for i in range(u):
+n = int(input("Podaj rozmiar lista: "))
+zestaw_2 = [random.randint(5, 15) for i in range(n)]
+print(zestaw_2)
 
